torch/csrc/distributed/c10d/ProcessGroupNCCL.hpp

```diff
@@ -816,7 +817,7 @@ class TORCH_API ProcessGroupNCCL : public Backend {
   std::mutex mutex_;
 
   // Heartbeat of watchdog thread.
-  uint64_t heartbeat_;
+  std::atomic_uint64_t heartbeat_;
```

Comment:
what is the difference between std::atomic<uint64_t> and std::atomic_uint64_t?

Reply:
no difference. just syntax sugar.